src/scripts/galleryTabs.js

```diff
@@ -0,0 +1,288 @@
+    }
+    for (var l = 0; l < dataHolder[0].stars; l++) {
+      sliderStars[l].className += " full"
+    }
+    sliderImages[0].classList.remove("js-image-ovl");
+  }
+
+  function imageSelecter() {
+    activeImages();
+    var sliderImages = slider.querySelectorAll(".js-image");
+    for (var i = 0; i < sliderImages.length; i++) {
+      (function (index) {
+        sliderImages[i].onclick = function () {
+          ovlImage.style.backgroundImage = "url('" + dataHolder[index].src + "')";
+          ovlImage.querySelector(".js-title").innerHTML = dataHolder[index].name;
+          ovlImage.querySelector(".price-new").innerHTML = dataHolder[index].newPrice;
+          ovlImage.querySelector(".price-old").innerHTML = dataHolder[index].oldPrice;
+          for (var j = 0; j < sliderImages.length; j++) {
+            sliderImages[j].className += " js-image-ovl";
+          };
+          this.classList.remove("js-image-ovl");
+          for (var k = 0; k < sliderStars.length; k++) {
+            sliderStars[k].classList.remove("full");
+          }
+          for (var l = 0; l < dataHolder[index].stars; l++) {
+            sliderStars[l].className += " full"
+          }
+        }
+      })(i);
+    }
+  }
+
+  function tabChecker() {
+    if (productsTabs[0].classList.contains("active")) {
+      topFeatures();
+      topFeatures(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      topFeatures(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      topFeatures(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      imageSelecter();
+      sliderLength();
+    } else if (productsTabs[1].classList.contains("active")) {
+      topRatedChecker(); //nie posiadamy info o liczbie sprzedazy, wiec na ta chwile uzyje funkcji z najlepszymi ocenami
+      topRatedChecker(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      topRatedChecker(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      topRatedChecker(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      imageSelecter();
+      sliderLength();
+    } else if (productsTabs[2].classList.contains("active")) {
+      topFeatures();
+      topFeatures(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      topFeatures(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      topFeatures(); //funkcja dodana ponownie, by sprawdzic dzialanie na wiekszej ilosci danych, pozniej zostanie usunieta
+      imageSelecter();
+      sliderLength();
+    } else if (productsTabs[3].classList.contains("active")) {
+      topRatedChecker();

```

Comment:
zamiast 0 1 2 3 może for po długości i if w środku - dla parzystych topFeatures();, a dla nieparzystych topRatedChecker(); i break, żeby przerwać pętle, jak wejdzie do if'a

Reply:
srednio to widze.... mam tu 3 rozne funkcje wykorzystywane w zaleznosci od aktywnej zakladki. docelowo powinny byc 4